test/spec/services/wfspermalink.spec.js

```diff
@@ -0,0 +1,146 @@
+goog.require('ngeo.WfsPermalink');
+goog.require(' ol.format.ogc.filter');
```

Comment:
This fixed the error, now I have another problem related to a rebase. But I will look into that next week.